_posts/2019-11-01-snowplow-r117-biskupin/index.md

```diff
@@ -0,0 +1,247 @@
+
+For example to start up an ssl-enabled, auto-upgrade server, following config can be used, collector configuration should contain:
+```json
+ssl {
+  enable = true
+  redirect = true
+  port = 443
+}
+```
+However, this configuration will use environment-defined JVM-attached certificates. In order to override the default behaviour and use a custom certificate, the low-level section can be defined as (akka config section):
+```json
+ssl-config {
+  keyManager = {
+    stores = [
+      {type = "PKCS12", classpath = false, path = ${CERT_FILE}, password = "pass" }
+    ]
+  }
+}
+```
+
+## IPv6 anonymization
+
+A new version of the Snowplow Common Enrich can be found on [Maven repository](https://mvnrepository.com/artifact/com.snowplowanalytics/snowplow-common-enrich_2.11/0.38.0)
+
+The schema for the configuration of the enrichment has been updated to version `1-0-1`:
+
+```json
+{
+	"schema": "iglu:com.snowplowanalytics.snowplow/anon_ip/jsonschema/1-0-1",
+	"data": {
+		"name": "anon_ip",
+		"vendor": "com.snowplowanalytics.snowplow",
+		"enabled": true,
+		"parameters": {
+			"anonOctets": 1,
+			"anonSegments": 1
+		}
+	}
+}
+```
+
+## Additional event fingerprint hashing methods
+
+A new version of the Snowplow Common Enrich can be found on [Maven repository](https://mvnrepository.com/artifact/com.snowplowanalytics/snowplow-common-enrich_2.11/0.38.0)
+
+
+The schema for the configuration of the enrichment has been updated to version `1-0-1`:
+
+```json
+{
+  "schema": "iglu:com.snowplowanalytics.snowplow/event_fingerprint_config/jsonschema/1-0-1",
+  "data": {
+    "name": "event_fingerprint_config",
+    "vendor": "com.snowplowanalytics.snowplow",
+    "enabled": true,
+    "parameters": {
+      "excludeParameters": ["cv", "eid", "nuid", "stm"],
+      "hashAlgorithm": "SHA1"
+    }
+  }
+}
+```
+
+## Support to the spot market for core instances
+
+A new version of the EmrEtl Runner can be found on [our Bintray repository](https://bintray.com/snowplow/snowplow-generic/snowplow) under `r117-biskupin` version.
+
+In order to enable spot instances, add a `core_instance_bid` setting to your `config.yml` file. This setting specifies a bid for an hour of EC2 spot instance in USD.
+
+```yaml
+aws:
+  emr:
+    jobflow:
+      core_instance_bid: 0.3
+```
+
+## Beam Enrich 
+
+A new version of the Beam Enrich can be found on [our Docker Hub repository](https://hub.docker.com/r/snowplow/beam-enrich/tags) under `0.4.0` tag.
+
+It contains the newest Snowplow Common Enrich.
+
+## Stream Enrich 
+
+A new version of the Stream Enrich can be found on [our Docker Hub repository](https://hub.docker.com/r/snowplow/stream-enrich-kinesis/tags) under `0.22.0` tag.
+
+It contains the newest Snowplow Common Enrich.
+
+## Spark Enrich
+
+Batch pipelines hasn't been extensively tested as part of our plans to focus on real-time and deprecate batch processing.
```

Comment:
Can you add a link to the the batch pipeline deprecation RFC, just afraid somebody still doesn't know about that.